src/content/developers/tutorials/how-to-mint-an-nft.md

```diff
@@ -0,0 +1,275 @@
+
+# How to Mint an NFT (Part 2/3 of NFT Tutorial Series)
+
+This tutorial describes how to mint an NFT on the Ethereum blockchain using our smart contract and Web3. You should have already completed Part 1: How to Write & Deploy an NFT, prior to starting Part 2 below. In Part 3, we will explain how to view your freshly minted NFT on Metamask!
```

Comment:
Would add link here for `Part 1: How to Write & Deploy an NFT` to go to part 1 tutorial. Same with `Part 3`